codalab/lib/worksheet_util.py

```diff
@@ -64,11 +64,15 @@
 WORKSHEET_ITEM_TYPES = (TYPE_MARKUP, TYPE_DIRECTIVE, TYPE_BUNDLE, TYPE_WORKSHEET)
 
 
-BUNDLE_REGEX = re.compile('^\s*(\[(.*)\])?\s*\{([^{]*)\}\s*$')
-SUBWORKSHEET_REGEX = re.compile('^\s*(\[(.*)\])?\s*\{\{(.*)\}\}\s*$')
+BUNDLE_REGEX = re.compile(
+    '^\s*(\\[(.*)\\])?\s*\\{([^{]*)\\}\s*$'  # NOQA - modifying escapes could create end-to-end test failure
```

Comment:
I think this is wrong formatting according to flake8. See my comment [here](https://github.com/codalab/codalab-worksheets/pull/2844#discussion_r495536795).